### PR TITLE
Implemented post blocks method

### PR DESCRIPTION
**Pull request by nishp77**
We could write tests for this? #37

### CODE REVIEW

**Comment by sno2** [comment]
@nishp77 I was planning on creating batch PRs for like 15,000+ coins for each of the class tests soon.

**Comment by nishp77** [comment]
please don't merge yet

File: src/bank.ts [hunk 1]
```diff
@@ -48,7 +48,18 @@ export class Bank extends ServerNode {
...
+  /**
+   * Adds new transaction blocks to the blockchain.
+   * @param balanceLock the current balance lock
+   * @param transactions the transactions to push to the block chain
+   * @param account the account that is sending the transactions
+   */
+  async addBlocks(balanceLock: string, transactions: Transaction[], account: Account) {
+    return await this.postData(
+      "/blocks",
+      { account, balanceLock, transactions }
```
Comment (sno2) [inline src/bank.ts:60]: For this, you should be setting the data in `postData` to `account.createSignedMessage` with the `balanceLock` and `transactions` as parameters to the `createSignedData` method.

**Review by sno2 — CHANGES_REQUESTED** [review]
Sorry, just saw your message to not review yet.

File: src/bank.ts [hunk 2]
```diff
@@ -57,7 +57,7 @@ export class Bank extends ServerNode {
   async addBlocks(balanceLock: string, transactions: Transaction[], account: Account) {
     return await this.postData(
       "/blocks",
-      { account, balanceLock, transactions }
+      account.createSignedData({ balanceLock, transactions })
```
Comment (sno2) [inline src/bank.ts:60]: Oh my gosh, so sorry.  I meant to say use the `account.createBlockMessage` method which doesn't require the destructuring (just feed the arguments in that order).  Very sorry for the confusion.

**Review by sno2 — CHANGES_REQUESTED** [review]
Sorry for the confusion.

File: src/bank.ts (hunk 1 above)
Comment (nishp77) [reply src/bank.ts]: i did use initially message but then you said data so i changed it, no worries

**Review by sno2 — APPROVED** [review]
LGTM, thanks.  Please reply in the PR with your account number so that we can pay you.  Sorry for the confusion.  And please make sure you run the `npm run prettier` command before you commit.  I can fix it, though👍

**Comment by nishp77** [comment]
c81061eb20ee1379ab98b6f5a125ebd0955278e36513eabd8d319967bb61d7e8

**Pull request merged**